pkg/api/customization/roletemplatebinding/validator.go

```diff
@@ -55,8 +55,13 @@ func (v *validator) validator(request *types.APIContext, schema *types.Schema, d
 		return nil
 	}
 
-	hasUserTarget := data["userId"] != nil || data["userPrincipalId"] != nil
-	hasGroupTarget := data["groupId"] != nil || data["groupPrincipalId"] != nil
+	userID, _ := data["userId"].(string)
```

Comment:
This will panic if interface is nil: https://play.golang.com/p/tncQH0k_qjh

Reply:
This is safe because it is doing the safety check. I am just throwing away the ok, but userID will get the zero value if the field is nil. See: https://play.golang.com/p/trIe8GXwDMV